src/libbson/fuzz/fuzz_test_bson_utf8_escape_for_json.c

```diff
@@ -0,0 +1,16 @@
+#include <bson/bson.h>
+
+int
+LLVMFuzzerTestOneInput (const uint8_t *data, size_t size)
+{
+   // Check if input crashes program (ok if return NULL)
+   char *got = bson_utf8_escape_for_json ((const char *) data, (ssize_t) size);
+
+   // If UTF-8 is valid, we should get some non-null response
+   if (bson_utf8_validate ((const char *) data, size, false)) {
```

Comment:
```suggestion
   if (bson_utf8_validate ((const char *) data, size, true)) {
```
Allow for escaped null characters in UTF-8 strings given `bson_utf8_escape_for_json` is capable of supporting it.